packages/ui/common/src/lib/components/json-view/json-view.component.html

```diff
@@ -16,7 +16,7 @@
   <div class="code-area">
     <pre *ngIf="highlight" class="line-numbers thin-scrollbars  ap-pb-1 "><code class="language-js"
       [innerHTML]="_content | outputLog"></code></pre>
-    <pre *ngIf="!highlight" [innerHTML]="_content | outputLog" class="ap-px-4 thin-scrollbars ap-py-3 ap-break-all">
+    <pre *ngIf="!highlight" [innerHTML]="_content | outputLog" class="ap-px-4 thin-scrollbars ap-py-3 wrap-text">
```

Comment:
I removed ap-break-all, I am assuming it was supposed to the wrap-text but it wasnt working so to avoid conflicts in the future I took it out.